index.html

```diff
@@ -30,16 +59,16 @@ <h2>The Past</h2>
             <h2>Why Did It Have To Be Boxes...</h2>
             
             <div class="boxes">
-                <div class="box">Box 1</div>
-                <div class="box">Box 2</div>
-                <div class="box">Box 3</div>
-                <div class="box">Box 4</div>
-                <div class="box">Box 5</div>
-                <div class="box">Box 6</div>
-                <div class="box">Box 7</div>
-                <div class="box">Box 8</div>
-                <div class="box">Box 9</div>
-                <div class="box">Box 10</div>
+                <div class="box1">Box 1</div>
+                <div class="box2">Box 2</div>
+                <div class="box3">Box 3</div>
+                <div class="box4">Box 4</div>
+                <div class="box5">Box 5</div>
```

Comment:
One of the main principles of programming is writing code that is reusable. In this case, for each of the boxes you want all the boxes the same size, but you want them different colors. One of the ways you could achieve this is by keeping the "box" class that assigns that specific element its shape and add an additional class to each box to specify the color of the box. For example:

```
<div class="boxes">
    <div class = "box box1">Box 1</div>
    <div class = "box box2">Box 2</div>
    <div class = "box box3">Box 3</div>
</div>
```
This allows you to reuse your CSS styles for the "box" class, while "box X" is the specified color.
Nonetheless, great attempt!